examples/worlds/link_velocity_controller.sdf

```diff
@@ -0,0 +1,346 @@
+            <link name="lower_link">
+                <pose>0.25 1.0 2.1 -2 0 0</pose>
+                <self_collide>0</self_collide>
+                <inertial>
+                <pose>0 0 0.5 0 0 0</pose>
+                </inertial>
+                <visual name="vis_lower_joint">
+                <pose>0 0 0 0 1.5708 0</pose>
+                <geometry>
+                    <cylinder>
+                    <radius>0.08</radius>
+                    <length>0.3</length>
+                    </cylinder>
+                </geometry>
+                <material>
+                    <ambient>0.8 0.8 0.8 1</ambient>
+                    <diffuse>0.8 0.8 0.8 1</diffuse>
+                    <specular>0.8 0.8 0.8 1</specular>
+                </material>
+                </visual>
+                <visual name="vis_cylinder">
+                <pose>0 0 0.5 0 0 0</pose>
+                <geometry>
+                    <cylinder>
+                    <radius>0.1</radius>
+                    <length>0.9</length>
+                    </cylinder>
+                </geometry>
+                <material>
+                    <ambient>0.8 0.8 0.8 1</ambient>
+                    <diffuse>0.8 0.8 0.8 1</diffuse>
+                    <specular>0.8 0.8 0.8 1</specular>
+                </material>
+                </visual>
+                <collision name="col_lower_joint">
+                <pose>0 0 0 0 1.5708 0</pose>
+                <geometry>
+                    <cylinder>
+                    <radius>0.08</radius>
+                    <length>0.3</length>
+                    </cylinder>
+                </geometry>
+                </collision>
+                <collision name="col_cylinder">
+                <pose>0 0 0.5 0 0 0</pose>
+                <geometry>
+                    <cylinder>
+                    <radius>0.1</radius>
+                    <length>0.9</length>
+                    </cylinder>
+                </geometry>
+                </collision>
+            </link>
+            <!-- pin joint for upper link, at origin of upper link -->
+            <joint name="upper_joint" type="revolute">
+                <parent>base</parent>
+                <child>upper_link</child>
+                <axis>
+                <xyz>1.0 0 0</xyz>
+                </axis>
+            </joint>
+            <!-- pin joint for lower link, at origin of child link -->
+            <joint name="lower_joint" type="revolute">
+                <parent>upper_link</parent>
+                <child>lower_link</child>
+                <axis>
+                <xyz>1.0 0 0</xyz>
+                </axis>
+            </joint>
+
+            <plugin
+                filename="ignition-gazebo-link-velocity-controller-system"
+                name="ignition::gazebo::systems::LinkVelocityController">
+                <link1_name>upper_link</link1_name>
+                <link2_name>lower_link</link2_name>
```

Comment:
this should be able to accept duplicate sdf elements in a plugin: i.e.
```
            <plugin
                filename="ignition-gazebo-link-velocity-controller-system"
                name="ignition::gazebo::systems::LinkVelocityController">
                <link_name>upper_link</link_name>
                <link_name>lower_link</link_name>
            </plugin>
```

we just need to parse it in a `while` loop, e.g. https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo4/src/systems/diff_drive/DiffDrive.cc#L173

As for subscribing to the cmds in the plugin, I noticed that only `<link2_name>` is parsed? For the above `while` approach, you can either create a callback for each link, or use a version of `Subscribe` with callback that gives you `MessageInfo` so you can figure out what topic that the callback is triggered for, e.g [like here](https://github.com/ignitionrobotics/ign-transport/blob/ign-transport9/test/integration/twoProcsPubSub.cc#L63)